lib/page.dart

```diff
@@ -46,6 +47,7 @@ class _FwupdPageState extends State<FwupdPage> {
     final model = context.watch<FwupdModel>();
     return Scaffold(
       appBar: AppBar(
+        toolbarHeight: 48,
```

Comment:
is this something that could be done in yaru?

Reply:
I'll try :) thanks

Reply:
done! :)